src/ask.rs

```diff
@@ -285,11 +285,14 @@ fn ask_on_brotherhood(
         .set("size", file_size::fit_4(dup_set.file_len));
     skin.print_owning_expander(&expander, &TextTemplate::from(MD_BROTHERHOOD));
     let mut q = Question::new("What do you want to do with these duplicates?");
-    for (i, &f) in brotherhood.files.iter().enumerate() {
-        let file_name = &dup_set.files[f].path.file_name().unwrap();
+    let mut names: Vec<(String, usize)> = brotherhood.files.iter()
+        .map(|&f| (dup_set.files[f].path.file_name().unwrap().to_string_lossy().to_string(), f))
```

Comment:
Maybe just define a simple struct (can be in the function) so that the semantics is clearer than with a tuple ?